import_owmat.py

```diff
@@ -135,13 +136,18 @@ def process_material(material, prefix, root, t):
         nodeTex = nodes.new('ShaderNodeTexImage')
         nodeTex.location = (-tile, -tile*(i))
         nodeTex.width = 250
-        nodeTex.color_space = 'NONE'
         
         tex = load_textures(texData[0], root, t)
         if tex is None:
             print('[import_owmat]: failed to load texture: {}'.format(texData[0]))
             continue
         nodeTex.image = tex.image
+        if nodeTex.image:
+            nodeTex.image.colorspace_settings.name = 'Non-Color'
+            
+            # Eye AO and Cornea materials should use "Alpha Blend" blend mode. Since shader type doesn't reliably correlate to alpha blend mode, we're identifying them by whether they use a specific texture. Not ideal but good enough for now.
+            if(nodeTex.image.name in ['0000000027EB', '0000000027ED']):
```

Comment:
There are hundreds of transparent and refractive transparent textures and materials in Overwatch, it's "not good enough" for now.